onnxruntime/core/framework/execution_frame.cc

```diff
@@ -375,6 +377,8 @@ Status ExecutionFrame::AllocateMLValueTensorSelfOwnBufferHelper(OrtValue& ort_va
     TraceAllocate(ort_value_index, size);
   }
 
+  dynamic_activation_memory_in_bytes_ += size;
```

Comment:
>dynamic_activation_memory_in_bytes_ [](start = 2, length = 35)

Do we need to track release?

Reply:
We don't. `dynamic_activation_memory_in_bytes_` is total allocated bytes during one batch.

---
In reply to: [440499958](https://github.com/microsoft/onnxruntime/pull/4241#discussion_r440499958) [](ancestors = 440499958)